openprescribing/pipeline/runner.py

```diff
@@ -441,16 +437,13 @@ def run_all(year, month, under_test=False):
     )
 
     activity = random.choice(
-        ["Put the kettle on", "Have a glass of wine", "Get up and stretch"]
+        ["put the kettle on", "have a glass of wine", "get up and stretch"]
     )
 
     msg = """
 Importing data for {}_{} complete!'
 
-You should now:
-* Tweet about it
-* Commit the changes to the smoke tests
-* {}
+You should now tweet about it and then {}
```

Comment:
Should we add something about "ask someone to restart the app server"?

Reply:
Yes good idea.